fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -424,7 +423,7 @@ protected void dropSchema() {
                     JdbcTarget target = new JdbcTarget(c);
                     IDatabaseAdapter adapter = getDbAdapter(dbType, target);
 
-                    if (this.dropSchema) {
+                    if (dropFhirSchema || dropOauthSchema || dropJavaBatchSchema) {
```

Comment:
is the adminDrop implicit here? what if we just call dropAdmin

Reply:
dropAdmin is covered separately in the if case just below this

Reply:
I think we should (later) revisit the whole option control logic in this Main...too much entropy.

Reply:
+1 on Robin's comment.  it's definitely growing